### PR TITLE
test(file-input): added more tests

### LINKED ISSUE #783: Enhance Test Coverage for `@yamada-ui/file-input`
related: #775

### Description
Enhance test coverage for components with less than 80% coverage to meet or exceed 80%.

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### PR DESCRIPTION
**Pull request by prathamnagpure**
Closes #783 

## Description

Added more tests for file-input

## Current behavior (updates)

> Statements, branch, functions, and lines have test coverage 73.68,  56.52, 62.5 and 78.12 percent respectively in the file-input component

## New behavior

> Statements, branch, functions, and lines have test coverage 100,  100, 100 and 100 percent respectively in the file-input component.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by hirotomoyamada** [comment]
@prathamnagpure 

Hi, thanks for the PR.

Please delete file in `.changeset`.

The `.changeset` files are needed when the package changes and are not needed for testing.

**Comment by prathamnagpure** [comment]
deleted it

File: packages/components/file-input/tests/file-input.test.tsx [hunk 1]
```diff
@@ -96,4 +96,193 @@ describe("<FileInput />", () => {
...
+
+    expect(setFiles).toHaveBeenCalledTimes(1)
+    expect(files).toBeTruthy()
+    if (files) {
+      expect(files.length).toEqual(3)
+      files.forEach((file: File) => expect(file).toEqual(mockedFile))
+    }
+  })
+
+  test("should not not show files in the onChange function if no files are present", () => {
```
Comment (setodeve) [inline packages/components/file-input/tests/file-input.test.tsx:166]: It might be better to use only one "not".

**Review by setodeve — CHANGES_REQUESTED** [review]

File: packages/components/file-input/tests/file-input.test.tsx (hunk 1 above)
Comment (prathamnagpure) [reply packages/components/file-input/tests/file-input.test.tsx]: I changed it, sorry for that.

**Review by setodeve — APPROVED** [review]
LGTM

**Pull request merged**